--- pcap-0.7.0/Cargo.toml	2019-04-09 18:42:37.312162320 +0530
+++ pcap-0.7.0/Cargo.toml	2019-04-09 18:43:35.597614728 +0530
@@ -36,6 +36,9 @@
 # This is disabled by default, because it depends on a tokio and mio
 tokio = ["mio", "tokio-core", "futures"]

+# This feature enables access to futures.
+futures-feature = ["futures"]
+
 # A shortcut to enable all features.
 full = ["pcap-savefile-append", "pcap-fopen-offline-precision", "tokio"]

@@ -65,3 +68,4 @@
 [[example]]
 name = "streamlisten"
 path = "examples/streamlisten.rs"
+required-features = ["futures-feature"]
